mercantile/__init__.py

```diff
@@ -425,7 +487,7 @@ def children(*tile, **kwargs):
     return tiles
 
 
-def simplify(*tiles):
+def simplify(tiles):
```

Comment:
@rohits2 please not3 that I changed this to have a single argument so that we can consume iterators as well as materialized sequences like lists.

Reply:
Do we need to do this to consume iterators? As far as I know, simplify can consume an iterator as long as it's passed as an object.

Reply:
good point: `simplify(*iter(tiles))` does materialize and then destructure, so it would be fine as is. Still, I think a single argument without the star is more "normal" and follows examples like https://docs.python.org/3/library/functools.html#functools.reduce.